### PR TITLE
Boards 7.3 and doc reorg

### PR DESCRIPTION
**Pull request by justinegeffen**
- Boards 7.3 content
- Doc reorg.

### CODE REVIEW

File: source/boards/groups-filter-sort.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+~~~~~~~~~~~~~~
+
+To ungroup cards on table view, select the **Group by** option at the top of the board, then select **Ungroup** from the top of the menu. This will return your table to its default state. Cards can be ungrouped in table view only. Ungrouping is not possible on board view since it requires groups to know what to display.
+
+Filters
+-------
+
+You can filter cards on your board if they utilize the **Select** property.
+
+Note: You must have at least one **Select** property on your board for filtering to work. Filtering by other property types such as **Person** or **Date** is not yet supported.
```
Comment (wuwinson) [inline source/boards/groups-filter-sort.rst:39]: Does not appear as a callout in preview mode.

File: source/boards/groups-filter-sort.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Add filters
+~~~~~~~~~~~
+
+To add a filter, select the **Filter** option at the top of the board, then select **+ Add filter**. The first **Select** property on your board will automatically be selected. To change the property to filter by, select the name of the first **Select** property, then select another property (if available) from the menu.
+
+**Specify the filtering criteria:**
+
+- **Includes**: Display cards with any of the specified values from the **Select** property.
+- **Doesn’t include**: Display all cards without any of the specified values from the “**Select** property.
```
Comment (wuwinson) [inline source/boards/groups-filter-sort.rst:49]:
```suggestion
- **Doesn’t include**: Display all cards without any of the specified values from the **Select** property.
```

File: source/boards/calculations.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
+Work with calculations
+======================
+
+When you view a board in table, Kanban, or board view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
```
Comment (wuwinson) [inline source/boards/calculations.rst:4]:
```suggestion
When you view a board in table or board view, you can use calculations to answer basic metric questions without needing to create complex reports. Hover over the bottom of a column to display the **Calculate** feature, then select the arrow to open the menu options.
```

Comment (wuwinson) [reply source/boards/calculations.rst:4]: Kanban and board view are the same

File: source/boards/share-and-collaborate.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Board permissions
+-----------------
+
+From Mattermost Boards v7.2, boards belong to teams, and any member of that team can be granted access to a board. Prior to Mattermost Boards v7.2, boards are tied to channel workspaces, and board membership is determined by channel membership.
+
+Roles
+-----
+
+The level of access to a board is determined by a user’s assigned board role.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:15]: Decided against using the table?

File: source/boards/share-and-collaborate.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+- **Admin:** Can modify the board, its contents, and its permissions. By default, board creators are also admins of the board.
+- **Editor:** Can modify the board and its contents.
+
+Manage team access
+~~~~~~~~~~~~~~~~~~~
+
+Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a minimum board role for everyone on the team.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:23]:
```suggestion
Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a default board role for everyone on the team. 
```

File: source/boards/share-and-collaborate.rst [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+- **Admin:** Can modify the board, its contents, and its permissions. By default, board creators are also admins of the board.
+- **Editor:** Can modify the board and its contents.
+
+Manage team access
+~~~~~~~~~~~~~~~~~~~
+
+Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a minimum board role for everyone on the team. 
+
+Depending on the role selected, everyone on the team will have access to the board with a minimum of the permissions from the role selected. Users can get elevated permissions based on their individual board membership. The default team access for a newly created board is **None**, which means nobody on the team has access to the board.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:25]:
```suggestion
Depending on the role selected, everyone on the team will have access to the board with the permissions from the role selected. Users can get elevated permissions based on their individual board membership. The default team access for a newly created board is **None**, which means nobody on the team has access to the board.
```

File: source/boards/share-and-collaborate.rst [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Manage individual board membership
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only board admins manage user permissions on a board, including adding, changing, and removing members.
+
+To add individual users from the team as explicit members of the board, open the Share dialog on the board, search for individual team members, then assign a role to set their permissions for the board. The role for individual board members overrides any role specified for team access.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:32]:
```suggestion
To add individual users from the team as explicit members of the board, open the **Share** dialog on the board, search for individual team members, then assign a role to set their permissions for the board. The role for individual board members overrides any role specified for team access.
```

File: source/boards/share-and-collaborate.rst [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+- To change a board member’s role, open the **Share** dialog, select the role dropdown next to the user’s name, then select another role from the list.
+- To remove a member from a board, open the **Share** dialog, select the role dropdown next to the user’s name, then select Remove member.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:35]:
```suggestion
- To remove a member from a board, open the **Share** dialog, select the role dropdown next to the user’s name, then select **Remove member**.
```

File: source/boards/share-and-collaborate.rst [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+Channel role groups
+~~~~~~~~~~~~~~~~~~~
+
+Board admins can add a channel to a board to grant all its members Editor access. To do this, select **Share** in the top-right corner of the board, search for the channel name, and add it to the board as a user. The default role is Editor. Doing so also links the board back to the channel, where the board will appear on the channel RHS.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:40]:
```suggestion
Board admins can add a channel to a board to grant all its members Editor access. To do this, select **Share** in the top-right corner of the board, search for the channel name, and add it to the board as a user. The default role is Editor. Doing so also [links the board back to the channel](ADD URL to "Link a board to a channel"), where the board will appear on the channel RHS.
```

File: source/boards/share-and-collaborate.rst [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+To unlink the channel from the board, open the **Share** dialog, select the role dropdown next to the channel’s name, then select Unlink.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:42]:
```suggestion
To unlink the channel from the board, open the **Share** dialog, select the role dropdown next to the channel’s name, then select **Unlink**.
```

File: source/boards/share-and-collaborate.rst [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+~~~~~~~~~~~~~~~~~~~
+
+Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a minimum board role for everyone on the team. 
+
+Depending on the role selected, everyone on the team will have access to the board with a minimum of the permissions from the role selected. Users can get elevated permissions based on their individual board membership. The default team access for a newly created board is **None**, which means nobody on the team has access to the board.
+
+Manage individual board membership
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only board admins manage user permissions on a board, including adding, changing, and removing members.
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:30]:
```suggestion
Only board admins can manage user permissions on a board, including adding, changing, and removing members.
```

File: source/boards/share-and-collaborate.rst [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+Share a board internally
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To share a board with team members internally, select **Share** in the top-right corner of the board, then select **Copy link** from the **Share** tab below. Paste the copied link in a channel or direct message to share the board with other team members. Only team members who have permissions to the board will be able to open the board from the shared link.
+
+Share a board publicly
+~~~~~~~~~~~~~~~~~~~~~~
+
+Sharing boards publicly is disabled by default. This means that the Publish tab is not available from the Share dialog. To enable public board sharing:
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:61]:
```suggestion
Sharing boards publicly is disabled by default. This means that the **Publish** tab is not available from the **Share** dialog. To enable public board sharing:
```

File: source/boards/share-and-collaborate.rst [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+Cards can be linked and shared with team members directly on Mattermost Channels. When you share a link to a card within Channels, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+To share a card, you'll need to copy the card link first:
+
+- Open a card and select the options menu **(...)** at the top right of the card, then select **Copy link**.
+- Alternatively, you can open the board view and hover your mouse over any card to access the options menu **(...)** for the card and select **Copy link** from there.
+
+After you've copied the link, paste it into any channel or direct message to share the card. A preview of the card will display within the channel with a link back to the card on Boards.
+
+Import and export a board
```
Comment (wuwinson) [inline source/boards/share-and-collaborate.rst:90]: @justinegeffen This section should be moved to the "Migrate boards" section. There's also updated content for that as well that should be added there

File: source/boards/navigate-boards.rst [hunk 14]
```diff
@@ -0,0 +1,64 @@
+Navigate boards
+===============
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+  
+If you're using Mattermost 7.0, open boards using the App Bar. If you don't see the Apps Bar to the right of your screen, you can access boards using the product menu in the top left corner of Mattermost.
```
Comment (wuwinson) [inline source/boards/navigate-boards.rst:7]: Recommend updating or removing this section. If a channel doesn't have linked boards, then you can't open Boards directly from the Apps Bar.

File: source/boards/navigate-boards.rst [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+
+After creating categories, you can move your boards to those categories by opening the options menu **(...)** next to the board and selecting **Move To…** to select the category where you want the board to be moved.
+
+If you delete a category with boards in it, then those boards will return to the default **Boards** category.
+
+Categories are organized per-user, so you can arrange your boards under categories that make sense to you without impacting boards or categories for other users. If a board is moved to a custom category, then the board will appear under that category for you only. Other users who are members of the board will continue to see the board in their own categories.
+
+.. note::
+
+For users upgrading to Mattermost Boards v7.2 onwards:
```
Comment (wuwinson) [inline source/boards/navigate-boards.rst:24]: Suggest putting this in a callout or differentiate it from the main text above.

File: source/boards/navigate-boards.rst [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+Boards that you create after the migration won’t be linked to a workspace and will always appear under the default **Boards** category unless you move or hide the boards.
+
+Manage boards on the sidebar
+----------------------------
+
+In addition to moving boards to other categories, from the  options menu **(...)** next to each board name you can perform the following actions:
+
+- **Delete board**: If you are an admin of the board, you will see an option to delete the board. Deleting the board permanently removes the board from the sidebar of all board members.
+- **Duplicate board**: Creates a copy of the board and all the cards on the board. The duplicated board will appear under the same category as the original board.
```
Comment (wuwinson) [inline source/boards/navigate-boards.rst:37]:
```suggestion
- **Duplicate board**: Creates a copy of the board and all the cards on the board. The duplicated board will appear under the same category as the original board. Board members from the original board are not migrated to the new board.
```

**Review by wuwinson — CHANGES_REQUESTED** [review]
Mostly bolded a bunch of text

File: source/boards/share-and-collaborate.rst (hunk 4 above)
Comment (justinegeffen) [reply source/boards/share-and-collaborate.rst:15]: Still adding the table. :)

File: source/boards/share-and-collaborate.rst [hunk 17]
```diff
@@ -0,0 +1,88 @@
...
+
+To add individual users from the team as explicit members of the board, open the **Share** dialog on the board, search for individual team members, then assign a role to set their permissions for the board. The role for individual board members overrides any role specified for team access.
+
+- To change a board member’s role, open the **Share** dialog, select the role dropdown next to the user’s name, then select another role from the list.
+- To remove a member from a board, open the **Share** dialog, select the role dropdown next to the user’s name, then select **Remove member**.
+
+Channel role groups
+~~~~~~~~~~~~~~~~~~~
+
+Board admins can add a channel to a board to grant all its members Editor access. To do this, select **Share** in the top-right corner of the board, search for the channel name, and add it to the board as a user. The default role is Editor. Doing so also [links the board back to the channel](ADD URL to "Link a board to a channel"), where the board will appear on the channel RHS.
```
Comment (justinegeffen) [inline source/boards/share-and-collaborate.rst:40]:
```suggestion
Board admins can add a channel to a board to grant all its members Editor access. To do this, select **Share** in the top-right corner of the board, search for the channel name, and add it to the board as a user. The default role is Editor. Doing so also `links the board back to the channel <https://docs.mattermost.com/boards/navigate-boards.html#link-a-board-to-a-channel>`_, where the board will appear on the channel RHS.
```

**Comment by justinegeffen** [comment]
Thanks, @wuwinson, for the super-thorough review. I've addressed everything barring the table which I'll add shortly.

File: source/boards/overview.rst [hunk 18]
```diff
@@ -13,4 +13,30 @@ With Boards you can:
...
+When working with cards, you can manage properties, add descriptions, attach images, assign them to team members, mention team members, add comments, and so on.
+
+Access your boards
+------------------
+
+Open the Boards tab via the product menu in the top left corner of Mattermost, to view all the boards for your team. You can select the Boards icon in the Apps Bar to open the right-hand panel, and display boards linked to the channel or message that you're in.
+
+If you don't see the Apps Bar and your boards layout looks different to what's described, you may be using an older version of Mattermost. The navigation content in this documentation will not be aligned with your navigation, but the functionality described will be applicable across versions unless otherwise noted.
+
+Focalboard personal server and desktop
```
Comment (justinegeffen) [inline source/boards/overview.rst:37]: @wuwinson, I've added this based on some of the questions we've received about features that don't apply across the options. Can you think of any others to add?

File: source/boards/work-with-cards.rst [hunk 19]
```diff
@@ -4,38 +4,109 @@ Work with cards
...
+Delete a property
+-----------------
+
+To delete properties you no longer need, select the property, then choose **Delete**. You’ll be asked to confirm that you want to remove that property from every card on the current board.
+
+.. note:: 
+  
+  Properties are displayed in the order they were created and cannot be re-ordered at this time.
+
+Define a **Select** or **Multi-select** property
```
Comment (wuwinson) [inline source/boards/work-with-cards.rst:81]:
```suggestion
Define a "Select" or "Multi-select" property
```

Comment (wuwinson) [reply source/boards/work-with-cards.rst:81]: This header with bold text is showing up weird on the live preview. Maybe quotes will render better.

File: source/boards/work-with-cards.rst [hunk 20]
```diff
@@ -46,22 +117,18 @@ Card badges are a quick way to view card details without opening up a card. To a
...
-
-Share card previews
--------------------
-
-When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+You can include a team member on a card by `mentioning them on a card <https://docs.mattermost.com/channels/mention-people.html>`__ the same way you would in Channels. Mentions are supported in the `Comments<https://docs.mattermost.com/boards/work-with-cards.html#comment-on-a-card>`_ and `Description <https://docs.mattermost.com/boards/work-with-cards.html#card-descriptions>`_ sections within a card. The team member you mention will receive a direct message notification from the boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
```
Comment (wuwinson) [inline source/boards/work-with-cards.rst:131]:
```suggestion
You can include a team member on a card by `mentioning them on a card <https://docs.mattermost.com/channels/mention-people.html>`__ the same way you would in Channels. Mentions are supported in the `Comments <https://docs.mattermost.com/boards/work-with-cards.html#comment-on-a-card>`_ and `Description <https://docs.mattermost.com/boards/work-with-cards.html#card-descriptions>`_ sections within a card. The team member you mention will receive a direct message notification from the boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
```

File: source/boards/work-with-views.rst [hunk 21]
```diff
@@ -0,0 +1,37 @@
+Work with saved views
+=====================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+You can change boards views to adjust how your cards are represented. To add a new view to a board, from the board header, select the menu next to the current view name. Scroll down and select **+ Add view**, then select the new visualization you’d like to use.
```
Comment (wuwinson) [inline source/boards/work-with-views.rst:7]:
```suggestion
You can change board views to adjust how your cards are represented. To add a new view to a board, from the board header, select the menu next to the current view name. Scroll down and select **+ Add view**, then select the new visualization you’d like to use.
```

**Review by wuwinson — CHANGES_REQUESTED** [review]
@justinegeffen Submitting these changes for now. Still have to go through Share, Navigate, and Migrate tomorrow.

**Pull request merged**